app/scripts/app.config.js

```diff
@@ -32,22 +32,27 @@ angular.module('confRegistrationWebApp')
       vars: {
         development: {
           apiUrl: 'https://api.stage.eventregistrationtool.com/eventhub-api/rest/',
-          tsysEnvironment: 'staging'
+          tsysEnvironment: 'staging',
+          cookieDomain: 'localhost'
         },
         staging: {
           apiUrl: 'https://api.stage.eventregistrationtool.com/eventhub-api/rest/',
-          tsysEnvironment: 'production'
+          tsysEnvironment: 'production',
+          cookieDomain: 'stage.eventregistrationtool.com'
         },
         production: {
           apiUrl: 'https://api.eventregistrationtool.com/eventhub-api/rest/',
-          tsysEnvironment: 'production'
+          tsysEnvironment: 'production',
+          cookieDomain: 'eventregistrationtool.com'
```

Comment:
It's probably not important, but should this be:

`cookieDomain: 'eventregistrationtool.com'`

Reply:
My understanding is subdomains can read a parent domain's cookies but not the other way around.

I found some article that referred me to  https://scripts.cmbuckley.co.uk/ which I played with a bit. Seems like that's how it functions.

Reply:
Have you tested this to see that it resolves the issue https://jira.cru.org/browse/EVENT-544?